FALSEHOODS.md

```diff
@@ -5,6 +5,13 @@
 Given how ubiquitous phone numbers are and how long they've been around, it's
 surprising how many false assumptions programmers continue to make about them.
 
+1. **An individual has a phone number**
```

Comment:
I think this is a good point, and a good thing to start with. We should probably though make the "An individual has only one phone number" come straight after it, and move the "Phone numbers that are valid today" one down to after "Phone numbers cannot be re-used".

If you would like to do the reordering, feel free to do so and upload another patch, otherwise we'll do a separate PR with this new falsehood and the reordering in it.